### DISCUSSION TITLE
Is it easy to modify bluesky's display?

### DISCUSSION THREAD

**Discussion by LukeAMcSherry** [TUDelft-CNS-ATM/bluesky#454] [Q&A]
Hello! I'm working in a lab interested in using bluesky to run a few experiments. Is it easy to modify bluesky's display to display stuff our lab wants displayed? thanks!

**Comment by ProfHoekstra** [comment]
Dear Luke,

Yes, there are four ways to do this:

-         In scenario files (or the command window), you can use drawing commands, like CIRCLE, POLY, POLYLINE, LINE, BOX, TRAILS to draw any shapes in any colour on the radar window. The current features are connected to switches and can be switched off.

-         In the plug-in folder you can also see how you van program gui plugons. Check out the example

-         And it is all open source, you could even go down into the code fi you want and make your own fork.

-         You could also make your own GUI- client

Good luck,
     Jacco Hoekstra

From: Luke McSherry ***@***.***>
Sent: maandag 20 februari 2023 21:37
To: TUDelft-CNS-ATM/bluesky ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [TUDelft-CNS-ATM/bluesky] Is it easy to modify bluesky's display? (Discussion #454)


Hello! I'm working in a lab interested in using bluesky to run a few experiments. Is it easy to modify bluesky's display to display stuff our lab wants displayed? thanks!
